textdata_reader.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=attribute-defined-outside-init
+"""
+mimic-cxr 2.0.0 medical report preprocessor for the text that does extraction
+of findings and impression part of the text. Tokenize the sentence, lowercase
+all the characters and remove the word contains non-alphabetic characters
+"""
+import argparse
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+from forte.data.readers import PlainTextReader
+from forte.data.span import Span
+from forte.pipeline import Pipeline
+from forte.processors.nltk_processors import NLTKWordTokenizer
+from forte.processors.lowercaser_processor import LowerCaserProcessor
+from mimic.onto.mimic_ontology import Impression, Findings, FilePath
+from forte.data.multi_pack import MultiPack
+from forte.processors.base import MultiPackProcessor
+from forte.data.caster import MultiPackBoxer
+from ft.onto.base_ontology import Token
+from forte.processors.writers import PackNameJsonPackWriter
+from forte.data.selector import NameMatchSelector
+
+
+class FindingsExtractor(PackProcessor):
+    r"""A wrapper of fingings extractor.
```

Comment:
Typo: findings.

Why this is a wrapper? I think you can describe this as "A processor to extract the Finding session in the medical report"